README.md

```diff
@@ -79,7 +79,8 @@ go install github.com/auth0/auth0-cli/cmd/auth0@latest
    - **macOS**: `$ tar -xf auth0-cli_{version}_Darwin_{architecture}.tar.gz`
    - **Linux**: `$ tar -xf auth0-cli_{version}_Linux_{architecture}.tar.gz`
    - **Windows**: Extract `auth0-cli_{version}_Windows_{architecture}.zip` using your preferred method of choice
-3. Run `./auth0`
+3. Make sure that the `PATH` and `HOME` environment variables include the folder where the binary was extracted.
+4. Run `./auth0`
```

Comment:
Maybe we should remove the `./` in order to reflect the cli being in the `PATH` var now.

Reply:
@andychiare do you mind making this change?

Reply:
Done! @m3talsmith